acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -486,6 +533,18 @@ public void waitForValidators(int numberOfValidators) {
         });
   }
 
+  public void waitForValidatorWithCredentials(int validatorIndex, Bytes32 expectedCredentials) {
+    waitFor(
+        () -> {
+          Optional<BeaconState> maybeState = fetchHeadState();
```

Comment:
Fetching the whole state seems overkill for this. We should just get the specific validator with `/eth/v1/beacon/states/{state_id}/validators/{validator_id}`